lib/widgets/message_input.dart

```diff
@@ -69,49 +99,73 @@ final class MessageInput extends HookWidget {
             child: Divider(height: 1),
           ),
         ],
+        if (editMode.value == _EditMode.emoji)
+          EmojiPicker(
+            key: const Key('emojiPicker'),
+            textEditingController: messageInputController,
+            config: Config(
```

Comment:
The styling here doesn't seem to get along w/ dark mode too well: 
![barely readable white text on white background with everything else being dark-mode-styled](https://github.com/user-attachments/assets/4f20df13-28d8-49ca-8184-5a2cb37373b7)

Reply:
That's fixed in #84.